packages/plugin-commands-import/test/index.ts

```diff
@@ -41,6 +41,7 @@ const DEFAULT_OPTS = {
   userAgent: 'pnpm',
   useRunningStoreServer: false,
   useStoreServer: false,
+  ignoreScripts: true,
```

Comment:
I don't understand what this changes if the value of ignoreScripts is hard coded in src/import.ts

Reply:
Because  `pnpm import` only generate pnpm-lock.yaml but not install the `node_modules` directory, so i think it maybe  not need to execute the scripts... pls see https://github.com/pnpm/pnpm/issues/3750